sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,622 @@
-    get isClosed(): boolean;
-    isReceivingMessages(): boolean;
-    receiveDeferredMessage(sequenceNumber: Long): Promise<ServiceBusMessage | undefined>;
-    receiveDeferredMessages(sequenceNumbers: Long[]): Promise<ServiceBusMessage[]>;
-    receiveMessages(maxMessageCount: number, maxWaitTimeInSeconds?: number): Promise<ServiceBusMessage[]>;
-    get receiveMode(): ReceiveMode;
-    registerMessageHandler(onMessage: OnMessage, onError: OnError, options?: MessageHandlerOptions): void;
-    renewMessageLock(lockTokenOrMessage: string | ServiceBusMessage): Promise<Date>;
-    }
-
-export { RetryOptions }
+export { RetryOptions };
 
 // @public
 export interface RuleDescription {
-    action?: string;
-    filter?: string | CorrelationFilter;
-    name: string;
+  action?: string;
+  filter?: string | CorrelationFilter;
+  name: string;
 }
 
 // @public
 export interface RuleDetails {
-    action?: SqlAction;
-    createdOn: string;
-    filter?: SqlFilter | CorrelationFilter;
-    ruleName: string;
-    subscriptionName: string;
-    topicName: string;
+  action?: SqlAction;
+  createdOn: string;
+  filter?: SqlFilter | CorrelationFilter;
+  ruleName: string;
+  subscriptionName: string;
+  topicName: string;
 }
 
 // @public
 export interface RuleOptions {
-    action?: SqlAction;
-    filter?: SqlFilter | CorrelationFilter;
+  action?: SqlAction;
+  filter?: SqlFilter | CorrelationFilter;
 }
 
 // @public
 export interface SendableMessageInfo {
-    body: any;
-    contentType?: string;
-    correlationId?: string | number | Buffer;
-    label?: string;
-    messageId?: string | number | Buffer;
-    partitionKey?: string;
-    replyTo?: string;
-    replyToSessionId?: string;
-    scheduledEnqueueTimeUtc?: Date;
-    sessionId?: string;
-    timeToLive?: number;
-    to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
-    viaPartitionKey?: string;
+  body: any;
+  contentType?: string;
+  correlationId?: string | number | Buffer;
+  label?: string;
+  messageId?: string | number | Buffer;
+  partitionKey?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  scheduledEnqueueTimeUtc?: Date;
+  sessionId?: string;
+  timeToLive?: number;
+  to?: string;
+  userProperties?: {
+    [key: string]: any;
+  };
+  viaPartitionKey?: string;
 }
 
 // @public
 export class Sender {
-    cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
-    cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
-    close(): Promise<void>;
-    get isClosed(): boolean;
-    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo): Promise<Long>;
-    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[]): Promise<Long[]>;
-    send(message: SendableMessageInfo): Promise<void>;
-    sendBatch(messages: SendableMessageInfo[]): Promise<void>;
-    }
+  cancelScheduledMessage(sequenceNumber: Long, options?: OperationOptions): Promise<void>;
```

Comment:
.NET: ServiceBusReceiver
Java: ServiceBusAsyncReceiver and ServiceBusReceiver
JS: Sender
Python: Something.

Reply:
@KieranBrantnerMagee - @conniey displayed some disrespect here...

Reply:
@conniey - so actually...there is an SDK style rule that says we have to have the name of the service in the name of the interface/class, correct? Is that what you're following to derive your names?

@KieranBrantnerMagee , @yunhaoling - what is the name of these classes in Python? ServiceBus*something?